content/_data/changelogs/weekly.yml

```
@@ -9391,6 +9391,94 @@
   # pull: 5029 (PR title: Queue (mainly) style consistency)
   # pull: 5033 (PR title: JDK 11.0.9 / 8.0.272+ compatible FunctionsTest#printThrowable())
 
+- version: '2.266'
+  date: 2020-11-10
+  changes:
+  - type: major bug
+    category: major bug
+    pull: 5046
+    issue: 64040
+    authors:
+    - timja
+    message: |-
+      Fix hidden page elements from radio blocks appearing when they should not.
+  - type: rfe
+    category: rfe
+    pull: 5008
+    issue: 62446
+    authors:
+    - aHenryJard
+    message: |-
+      Make the Maven Configuration of Global Tools compatible with configuration as code plugin.
+  - type: rfe
+    category: rfe
+    pull: 4294
+    issue: 41891
+    authors:
+    - jsoref
+    message: |-
+      Minor English cleanup for resource root url cleanup.
+  - type: major rfe
+    category: developer
+    pull: 4944
+    issue: 19561
+    authors:
+    - jglick
+    message: |-
+      Developer: Jenkins now uses an updated version of the XStream serialization library without custom patches.
+  - type: major rfe
+    category: developer
+    pull: 4848
+    authors:
+    - jglick
+    message: |-
+      Developer: Jenkins now uses Spring Security rather than its predecessor, Acegi Security.
+      Other bundled Spring libraries are also updated.
+  - type: rfe
+    category: developer
+    pull: 4929
+    authors:
+    - fqueiruga
+    message: |-
+      Developer: Upgrade jQuery to 3.5.x.
```

```suggestion
      Developer: Upgrade jQuery from 2.1.4 to to 3.5.1.
```

Note that the best practice for changelogs in the PR template recommends linking changelogs of upstream dependencies

---

Included in 884344495b along with references to release notes.  Intentionally limited the number of release notes links to 3 even though this increments much more than 3 releases.  Didn't want to spend the time to find the older release notes.